project1/templates/login.html

```diff
@@ -48,6 +48,44 @@
         display: block;
         }
 </style>
+{% endblock %}
+{% block scripts%}
+<script type = "text/javascript"> 
+// Search Form
+function SearchApi( ) {
+  var xhr = new XMLHttpRequest();
+  var query = document.getElementById('query').value;
+  xhr.open("POST", "/api/search", true);
+  xhr.setRequestHeader("Content-type", "application/json");
+  let json = JSON.stringify ({
+    "query" : query
+  });
+  xhr.send(json);
+  xhr.onload = () => {
+    if (xhr.status == 200) {
+      var data = JSON.parse(xhr.responseText);
+      var htmltext = "<br><h1 id = 'display' style = 'text-align:center'>Books</h1><br><div class='container'>"+
+            "<div class='table-wrapper-scroll-y my-custom-scrollbar'>"+"<table xlass = 'table' border = 1 align = 'center' class='table table-striped'>"+
+            "<th class='bg-secondary text-white'> {{ 'ISBN' }}</th>"+"<th class='bg-secondary text-white'> {{ 'Title' }}</th>"+
+            "<th class='bg-secondary text-white'> {{ 'Author' }}</th>"+"<th class='bg-secondary text-white'> {{ 'Year' }}</th>"+
+            "<th class='bg-secondary text-white'> {{ 'View Book Page' }}</th>";
+            for(var i = 0; i < data.length; i++) {
+              htmltext = htmltext + "<tr>" + "<td>"+data[i]["isbn"] +"</td>"+ "<td>" + data[i]["title"] +"</td>"+
+              "<td>" + data[i]["author"] +"</td>"+ "<td>" + data[i]["year"] +"</td><td><button onclick = 'bookdisplay(this)' id = "+
+              data[i]["isbn"] + ">View Book</button></td></tr>"
+            }
+      htmltext = htmltext + " </tbody> </table> </div></div>"
+      document.getElementById("htmltext").innerHTML = htmltext;
+    } else if(xhr.status == 404) {
+      console.log("Error: Page Not Found");
```

Comment:
This error should be visible to the user right ?? console.log() won't solve the purpose. It just only displays in the console.

Reply:
https://github.com/Madhuri97/project-1-105/pull/10/commits/bd4a9c393d9defddc16c17495fb863034e168d1f